modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractDartCodegen.java

```diff
@@ -489,6 +490,16 @@ public String getTypeDeclaration(Schema p) {
 
     @Override
     public String getSchemaType(Schema p) {
+		if (p instanceof ComposedSchema) {
+			ComposedSchema cs = (ComposedSchema) p;
+            if (cs.getOneOf() != null) {
+				List<Schema> schemas = ModelUtils.getInterfaces(cs);
+				if(schemas.size() != 1) {
+					return "dynamic";
```

Comment:
We should go with `Object` or `Object?` for all new code, depending on if it is nullable/required.